pkg/pool-weighted/contracts/managed/ManagedPoolSettings.sol

```diff
@@ -853,75 +855,58 @@ abstract contract ManagedPoolSettings is BasePool, ProtocolFeeCache, ReentrancyG
      * The caller may additionally pass a non-zero `burnAmount` to burn some of their BPT, which might be useful
      * in some scenarios to account for the fact that the Pool now has fewer tokens. This is a permissioned function.
      * @param token - The ERC20 token to be removed from the Pool.
-     * @param recipient - The address to receive the Pool's balance of `token` after it is removed.
      * @param burnAmount - The amount of BPT to be burned after removing `token` from the Pool.
-     * @param minAmountOut - Will revert if the number of tokens transferred from the Vault is less than this value.
-     * @return The amount of tokens the Pool held, sent to `recipient`.
      */
     function removeToken(
         IERC20 token,
-        address recipient,
-        uint256 burnAmount,
-        uint256 minAmountOut
-    ) external authenticate nonReentrant whenNotPaused returns (uint256) {
-        // We require the pool to be initialized (shown by the total supply being nonzero) in order to remove a token,
-        // maintaining the behaviour that no exits can occur before the pool has been initialized.
-        // This prevents the AUM fee calculation being triggered before the pool contains any assets.
+        address sender,
```

Comment:
This allows the manager to burn BPT from unrelated addresses which is why we hardcoded this to `msg.sender`.

Reply:
They can also mint to any address via `addToken`, which is the same as they can dillute everyone. They can also add bad tokens and drain the pool. I don't think this changes the trust model in any way.

Reply:
It's a question about blanket vs targeted actions. Similar to DAI vs USDC, if you hold DAI then Circle could blacklist Maker and make any USDC-backed DAI worthless but they can't blacklist your wallet specifically.

Reply:
We agreed on this being an issue for the controller rather than the pool.

Reply:
We decided that this is ultimately a controller concern - how they use burn amount and sender is very much up to them, and part of their trust model.